src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -57,18 +54,39 @@ public CDSSubsystem() {
     CDSWheelControllerOne = new MotorController("Wheel Motor Controller 1", Constants.CDSWheelControllerOneID, 40);
     CDSWheelControllerTwo = new MotorController("Wheel Motor Controller 2", Constants.CDSWheelControllerTwoID, 40);
 
+    CDSWheelControllerOne.setInverted(true);
     CDSWheelControllerTwo.getSparkMax().follow(CDSWheelControllerOne.getSparkMax(), true);
 
-    colorSensors = new ColorSensorMuxed(0, 1, 2);
-    
-    //colorSensorOne = new ColorSensorV3(Constants.colorSensorPort1);
-    //colorSensorTwo = new ColorSensorV3(Constants.colorSensorPort2);
-    //backBeamBreak = new DigitalInput(Constants.initialBallSensorChannel);
+    CDSBeltController.setIdleMode(IdleMode.kBrake);
+    //CDSWheelControllerOne.setIdleMode(IdleMode.kCoast);
+
+    colorSensors = new ColorSensorMuxed(0, 1, 3);
 
     String allianceColor = DriverStation.getAlliance().toString();
     SmartDashboard.putString("Alliance Color", allianceColor);
   }
 
+  public void CDSToggleAll(boolean reverse) {
+    if (reverse){
+      CDSWheelControllerOne.getSparkMax().set(-Constants.CDSWheelControllerSpeed);
+      SmartDashboard.putString("CDS Wheel Direction", "Reverse");
+      SmartDashboard.putNumber("CDS Wheel Speed", -Constants.CDSWheelControllerSpeed);
+
+      CDSBeltController.getSparkMax().set(-Constants.CDSBeltSpeed);
+      CDSBeltController.setIdleMode(IdleMode.kBrake);
```

Comment:
We don't need to set this idle mode over and over -- let's save time and set it once when the MotorController is created.